src/main/java/koreatech/in/controller/admin/AdminUserController.java

```diff
@@ -85,6 +86,20 @@ ResponseEntity<User> getUser(@ApiParam(required = true) @PathVariable("id") int
         return new ResponseEntity<>(adminUserService.getUserForAdmin(id), HttpStatus.OK);
     }
 
+    @ApiOperation(value = "특정 학생 조회", authorizations = {@Authorization("Authorization")})
+    @ApiResponses({
+            @ApiResponse(code = 401, message = "- 잘못된 접근일 때 (code: 100001) \n" +
+                                               "- 액세스 토큰이 만료되었을 때 (code: 100004) \n" +
+                                               "- 액세스 토큰이 변경되었을 때 (code: 100005)", response = ExceptionResponse.class),
+            @ApiResponse(code = 403, message = "- 권한이 없을 때 (code: 100003)", response = ExceptionResponse.class),
+            @ApiResponse(code = 404, message = "- 조회한 회원이 존재하지 않을 때 (code: 101003)", response = ExceptionResponse.class)
+    })
+    @RequestMapping(value = "/admin/users/student/{id}", method = RequestMethod.GET)
+    public @ResponseBody
+    ResponseEntity<StudentResponse> getStudent(@ApiParam(name = "id", required = true) @PathVariable("id") int id) throws Exception {
```

Comment:
@daheeParkk 
```suggestion
    ResponseEntity<StudentResponse> getStudent(@ApiParam(required = true) @PathVariable("id") int id) throws Exception {
```
ApiParam에서는 PathVariable는 name을 사용하면 파라미터 이름과 다를 시 스웨거에서 URL에 변수를 매핑시키지 못합니다.... 따라서 PathVariable 파라미터에 ApiParam을 사용한 설명을 할 때는 value를 통해 진행하고, ApiParam의 name을 사용하는 경우는 RequestBody를 설명할 때로 제한하는 것이 바람직해 보입니다.